neo4j/src/main/java/org/hibernate/ogm/datastore/neo4j/remote/dialect/impl/RemoteNeo4jTupleAssociationSnapshot.java

```diff
@@ -64,8 +64,6 @@ else if ( targetNode.containsKey( targetColumnName ) ) {
 				else {
 					// Ex: @ElementCollection List<Embedded> examples
 
-					targetColumnName = targetColumnName.substring( collectionRole.length()  + 1 );
-
```

Comment:
I suspect is the same reason this has changed: https://github.com/hibernate/hibernate-ogm/pull/728/files#diff-378d3f25166f2e2a798f69b74262fe86L66

The column has already the right name and therfore we don't have remove the collectionrole from the string.